frontend/src/entities/player/ui/PlayerStatsCard.styles.ts

```diff
@@ -0,0 +1,63 @@
+import { Avatar, Box, Chip } from '@mui/material'
+import { grey } from '@mui/material/colors'
+
+import { styled } from '@mui/material/styles'
+
+export const CardContainer = styled(Box)({
+  display: 'flex',
+  flexDirection: 'column',
+  gap: '16px',
+  backgroundColor: grey[50],
+  border: '1px solid',
+  borderColor: grey[300],
+  borderRadius: '4px',
+  maxWidth: '1000px',
+  padding: '16px',
+})
+
+export const PlayerInfo = styled(Box)(({ theme }) => ({
+  display: 'flex',
+  justifyContent: 'space-between',
+  alignItems: 'flex-start',
+  [theme.breakpoints.down('sm')]: {
+    flexDirection: 'column-reverse',
+    gap: '8px',
+  },
+}))
+
+export const PlayerDetails = styled(Box)({
+  display: 'flex',
+  alignItems: 'flex-start',
+  gap: '16px',
+})
+
+export const PlayerImage = styled(Avatar)({
+  width: '40px',
+  height: '40px',
+  objectFit: 'contain',
+  borderRadius: '4px',
+})
+
+export const Info = styled(Box)({
+  display: 'flex',
+  flexDirection: 'column',
+})
+
+export const AffiliationContainer = styled(Box)(({ theme }) => ({
+  display: 'flex',
+  flexDirection: 'column',
+  alignItems: 'flex-end',
+  gap: '8px',
+  [theme.breakpoints.down('sm')]: {
+    alignItems: 'flex-start',
+  },
+}))
```

Comment:
```suggestion
export const AffiliationContainer = styled(Box)(({ theme }) => ({
  display: 'flex',
  flexDirection: 'column',
  alignItems: 'flex-start',
  gap: '8px',

  [theme.breakpoints.up('sm')]: {
    alignItems: 'flex-end',
  },
}))
```

в целом касательно стилей -- если делаются мобилки, то лучше придерживаться mobile-first

Reply:
[refactor: mobile-first styles refactor](https://github.com/IvanBelikov/fantasy_football/pull/2/commits/07b10fa0578972812ceff8d719e4e7e7ff3d20d3)